src/generator/models.ts

```diff
@@ -553,7 +553,7 @@ function generateAliasType(structDef: StructDef, receiver: string, forMarshal: b
     if (prop.schema.type === SchemaType.DateTime) {
       text += `\t\t${prop.language.go!.name} *${prop.schema.language.go!.internalTimeType} \`${structDef.Language.marshallingFormat}:"${sn}"\`\n`;
     } else if (prop.language.go!.isAdditionalProperties || prop.language.go!.needsXMLDictionaryUnmarshalling) {
-      text += `\t\t${prop.language.go!.name} *additionalProperties \`${structDef.Language.marshallingFormat}:"${sn}"\`\n`;
+      text += `\t\t${prop.language.go!.name} *map[string]string \`${structDef.Language.marshallingFormat}:"${sn}"\`\n`;
```

Comment:
This is incorrect.  The `additionalProperties` type is defined in a helper that's only emitted when `needsXMLDictionaryUnmarshalling` is true.  The fact that this is being emitted when the helper isn't suggests that the above condition has a bug.

Reply:
It's also interesting that this failed only for Go 1.14.  Do you know why that is?